spec/ruby-push-notifications/apns/apns_pusher_spec.rb

```diff
@@ -180,9 +180,8 @@ module APNS
             end
 
             it 'saves results' do
-              expect do
-                pusher.push notifications
-              end.to change { notifications.map { |n| n.results } }.from([nil]*10).to([[NO_ERROR_STATUS_CODE]]*10)
+              pusher.push notifications
+              expect(notifications.map { |n| n.individual_results }).to eq [[NO_ERROR_STATUS_CODE]]*10
```

Comment:
Surrounding space missing for operator '*'.<br>Pass `&:individual_results` as an argument to `map` instead of a block.